docs/src/components/home/DiamondSponsors.tsx

```diff
@@ -27,54 +27,87 @@ const DIAMONDs = [
 
 export default function DiamondSponsors() {
   return (
-    <Grid container spacing={{ xs: 2, md: 4 }}>
-      {DIAMONDs.map((item) => (
-        <Grid item key={item.name} xs={12} sm={6} md={4}>
-          <SponsorCard
-            inView
-            item={item}
-            bottom={<SponsorLabel color="primary">Diamond sponsor</SponsorLabel>}
-          />
-        </Grid>
-      ))}
-      <Grid item xs={12} sm={6} md={4}>
-        <Paper
-          variant="outlined"
+    <Box>
+      <Box sx={{mb: 2, display: 'flex', alignItems: 'center'}}>
+        <Box
           sx={{
-            p: 2,
-            display: 'flex',
-            alignItems: 'center',
-            height: '100%',
-            borderStyle: 'dashed',
+            display: 'inline-block',
+            mr: 1,
+            mt: 0.5,
+            borderRadius: 1,
+            width: 12,
+            height: 12,
+            bgcolor: (theme) =>
+              theme.palette.mode === 'dark'
+                ? theme.palette.primary[200]
+                : theme.palette.primary[500],
+            border: '3px solid',
             borderColor: (theme) =>
-              theme.palette.mode === 'dark' ? 'primaryDark.400' : 'grey.300',
+              theme.palette.mode === 'dark'
+                ? theme.palette.primary[500]
+                : theme.palette.primary[100],
           }}
-        >
-          <IconButton
-            aria-label="Become MUI sponsor"
-            component="a"
-            href="mailto:diamond@mui.com"
-            target="_blank"
-            rel="noopener noreferrer"
-            color="primary"
-            sx={{ mr: 2, border: '1px solid', borderColor: 'divider' }}
-          >
-            <AddRounded />
-          </IconButton>
-          <Box>
-            <Typography variant="body2" color="text.primary" fontWeight="bold">
-              Become our sponsor!
-            </Typography>
-            <Typography variant="body2" color="text.primary">
-              To join us, contact us at{' '}
-              <Link href="mailto:diamond@mui.com" target="_blank" rel="noopener noreferrer">
-                diamond@mui.com
-              </Link>{' '}
-              for pre-approval.
-            </Typography>
-          </Box>
-        </Paper>
-      </Grid>
-    </Grid>
+        />
+        <Typography
+          component="h3"
+          variant="h5"
+          fontWeight="extraBold"
+          sx={{color: (theme) =>
+            theme.palette.mode === 'dark'
+              ? theme.palette.primary[400]
+              : theme.palette.primary[500],}}>
+          Diamond
```

Comment:
@mbrookes 

>Diamond / gold footer on every card seems redundant, and makes the sponsors section larger than it needs to be. A heading might be better.

I have iterated on this a bit. Removed the footer on each card and added a heading for separating between Diamond and Gold. Although I feel like it loses a bit of character, it is indeed simpler and less redundant. Let me know what you think.

<img width="700" alt="Screen Shot 2021-08-24 at 16 21 34" src="https://user-images.githubusercontent.com/67129314/130677890-8a40b976-7370-4b95-b4b0-d8c15ed2f10e.png">

>Diamond sponsor logos (and possibly cards) should be bigger than gold

Regarding this, I need @siriwatknp help to make the `SponsorCard` component conditionally render the Avatar holding the logos in different sizes if the company is Diamond or Gold. I think we can make it just a tad bigger when Diamond for a better emphasis. 

> It might be intentional, but the external link icon moving on hover looks like a bug:
I've made just a small tweak so it's even more subtle. I don't personally feel like it looks like a bug. So, calling for more opinions here: @oliviertassinari let me know your thoughts.

Reply:
update avatar size in diamond to be `64px` (before was `40px`)

Reply:
@siriwatknp I have most of the logos (the only one missing is Movavi) in svg. I think we could switch to it instead of using pngs, some of them are of low quality, a little blurry. You can download the svgs in Figma! Let me know when you make the switch :)